install/streams.csv

```diff
@@ -884,10 +884,11 @@ JCWJ,20,200,true,false,true,2002-07-21T08:00:00Z,2004-05-27T19:00:00Z
 JCZ,10,1,false,false,false,2004-06-23T04:20:00Z,9999-01-01T00:00:00Z
 JCZ,10,100,false,false,false,2004-06-23T04:20:00Z,9999-01-01T00:00:00Z
 JCZ,20,1,false,false,false,2004-06-25T00:00:00Z,2018-11-27T00:00:00Z
-JCZ,20,50,false,false,false,2004-06-25T00:00:00Z,2020-01-15T22:54:30Z
-JCZ,20,200,false,false,true,2004-06-25T00:00:00Z,2020-01-15T22:54:30Z
-JCZ,21,50,false,false,false,2020-01-15T22:55:00Z,2020-06-16T00:00:00Z
-JCZ,21,200,false,false,true,2020-01-15T22:55:00Z,9999-01-01T00:00:00Z
+JCZ,20,50,false,false,false,2004-06-25T00:00:00Z,2018-11-27T00:00:00Z
+JCZ,20,200,false,false,true,2004-06-25T00:00:00Z,2018-11-27T00:00:00Z
+JCZ,21,50,false,false,false,2018-11-27T03:00:00Z,2020-06-16T00:00:00Z
+JCZ,21,200,false,false,true,2018-11-27T03:00:00Z,2020-08-30T23:59:59Z
+JCZ,21,200,false,false,false,2020-08-31T00:00:00Z,9999-01-01T00:00:00Z
```

Comment:
Hi @salichon 
A quick question: how do we know when to end the 200 triggered and start the 200 continuous? Where is this information based on?

Reply:
Ah, okey. I understand now. The data started streaming continuously on `2020.244 (Aug.31)`.

```
(base) murieln@ubuntu:~$ aws s3 ls --profile sod_readrole s3://geonet-archive/miniseed/2020/2020.244/JCZ.NZ/
2020-09-06 02:40:41      19968 2020.244.JCZ.01-ACE.NZ.D
2020-09-05 09:47:24      61952 2020.244.JCZ.01-LCQ.NZ.D
2020-09-06 02:40:40      20480 2020.244.JCZ.01-LOG.NZ.D
2020-09-05 09:47:23       6656 2020.244.JCZ.01-VEC.NZ.D
2020-09-05 09:47:23       6656 2020.244.JCZ.01-VEM.NZ.D
2020-09-05 09:47:18       6656 2020.244.JCZ.01-VEP.NZ.D
2020-09-05 09:47:17       6656 2020.244.JCZ.01-VKI.NZ.D
2020-09-05 09:47:16       4608 2020.244.JCZ.02-LEX.NZ.D
2020-09-05 09:47:15   10941952 2020.244.JCZ.10-HHE.NZ.D
2020-09-05 09:47:17   11325952 2020.244.JCZ.10-HHN.NZ.D
2020-09-05 09:47:17   10594304 2020.244.JCZ.10-HHZ.NZ.D
2020-09-05 09:47:14     211968 2020.244.JCZ.10-LHE.NZ.D
2020-09-05 09:47:14     213504 2020.244.JCZ.10-LHN.NZ.D
2020-09-05 09:47:12     207872 2020.244.JCZ.10-LHZ.NZ.D
2020-09-05 09:47:12       6656 2020.244.JCZ.10-VME.NZ.D
2020-09-05 09:47:06       6656 2020.244.JCZ.10-VMN.NZ.D
2020-09-05 09:47:06       6656 2020.244.JCZ.10-VMZ.NZ.D
2020-09-05 09:47:05    2618368 2020.244.JCZ.21-HNE.NZ.D
2020-09-05 09:47:05    2684928 2020.244.JCZ.21-HNN.NZ.D
2020-09-05 09:47:05    2512384 2020.244.JCZ.21-HNZ.NZ.D

```